src/app/personsok/Sokeresultat.tsx

```diff
@@ -17,11 +17,18 @@ function Sokeresultat(props: Props) {
     useFocusOnMount(ref);
 
     const tittelRekke = ['Fødselsnummer', 'Navn', 'Adresser', 'Bosted'];
+    for (let linje of props.response) {
```

Comment:
kan vi gjøre noe alà;
````
const trengerUtenlandskIdTittel = props.response
  .some((person) => person.utenlandskID?.utenlandskID.identifikasjons !== undefined);
if (trengerUtenlandskIdTittel) {
  tittelRekke.push('Utenlandsk ID');
}
```

Da slipper vi å loope gjennom hele responsen om første personen har `utenlandskID`

Reply:
Takk for tilbakemeldingen, har endret koden og testet med mock-data nå 😄